src/js/DrawStyleController.js

```diff
@@ -188,9 +213,20 @@ goog.require('ga_styles_service');
     $scope.options.color = options.colors[5];
     $scope.options.textColor = options.colors[5];
     $scope.options.textSize = options.textSizes[0];
-    $scope.options.icon = options.icons[0];
     $scope.options.iconColor = options.colors[5];
     $scope.options.iconSize = options.iconSizes[2];
+    $scope.options.iconCategory = options.iconCategories[0];
+
+    $scope.options.iconCategories.forEach(function(category) {
+      if (category.type === 'img') {
+        category.icons = getImgsByCategory(category);
+      } else {
+        category.icons = iconsCSS;
+      }
+    });
+
+    $scope.options.icons = $scope.options.iconCategories[0].icons;
+    $scope.options.icon = options.icons[0];
```

Comment:
Use  `$scope.options.icons[0]`   . It's more clear than options.icons, since we don't define this property directly on options object like before.